v21.2/alter-role.md

```diff
@@ -35,7 +35,7 @@ table td:first-child {
 
 Parameter | Description
 ----------|-------------
-`role_name` | Specify the name of the role that you want to alter.
+`role_name` | The name of the role to alter.
```

Comment:
Would also be nice to hold to style PR in future.  Different brain required for this than reading new content.  Harder to switch between them